clients/usage-logging/src/test/integration/UsageLoggingClient.test.ts

```diff
@@ -16,10 +16,17 @@ describe("UlasClient - OIDC Token (#integration)", () => {
   let accessToken: AccessToken;
 
   before(async () => {
+    if (process.env.IMJS_OIDC_ULAS_TEST_CLIENT_ID === undefined)
+      throw new Error("Could not find IMJS_OIDC_ULAS_TEST_CLIENT_ID");
+    if (process.env.IMJS_OIDC_ULAS_TEST_REDIRECT_URI === undefined)
+      throw new Error("Could not find IMJS_OIDC_ULAS_TEST_REDIRECT_URI");
+    if (process.env. IMJS_OIDC_ULAS_TEST_SCOPES === undefined)

```

Comment:
unnecessary space between env. and IMJS_

Reply:
Good catch, thanks.